src/index.html

```diff
@@ -8,6 +8,73 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="block">
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+        
+        <div class="stars__star"></div>
+        
+        <div class="stars__star"></div>
+        
+        <div class="stars__star"></div>
+        
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--1">
+        <div class="stars__star star_active"></div>
```

Comment:
Due to the task description you dont need to add additional classes like star__active
![image](https://user-images.githubusercontent.com/80700247/229531585-8b4488a0-93d8-4539-bbf9-f01265d14f19.png)

Instead you can use :nth-child to set appropriate image